### DISCUSSION TITLE
Backstage TechDocs Issue: rendering code annotations

### DISCUSSION THREAD

**Discussion by ghost** [squidfunk/mkdocs-material#5774] [General]
Hello! 

The [Backstage](https://backstage.io/) project has a feature called [TechDocs](https://backstage.io/docs/features/techdocs/) that integrates documentation and uses the mkdocs material theme. 
To leverage TechDocs, you need to install the `techdocs-core` plugin for mkdocs. 
Installing this plugin breaks the rendering of code annotations.

This issue is described in [#128](https://github.com/backstage/mkdocs-techdocs-core/issues/128).

I've created a reproducible example and described some findings in [steven-terrana-bah/techdocs-core-128](https://github.com/steven-terrana-bah/techdocs-core-128). 

Truth be told - i'm not sure if the root cause of this issue is the `techdocs-core` plugin or if it's introducing an edge-case that breaks the mkdocs material theme. 

I'm happy to continue digging into this - but i'm looking for some clues on where I should dig in next to try and make progress.

**Comment by squidfunk** [comment]
Thanks for reporting. Since techdocs is a downstream project that we don't officially support, we can't put time into investigating. If you dig deeper and find an easy fix that doesn't break anything for our users, we might merge it if we consider it safe. Please understand that our time to maintain an Open Source project is limited, and while we have substantial funding, we need to invest it into pushing this very project forward 😊

Clues: code annotations are entirely implemented in JavaScript on the client side. Source code is located here:
https://github.com/squidfunk/mkdocs-material/tree/master/src/assets/javascripts/components/content/annotation

**Reply by ghost** [reply]
Of course. Thanks for the clue! And your work on this project :)
